<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AdminWindow</class>
 <widget class="QMainWindow" name="AdminWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>764</width>
    <height>364</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="admin_name_label">
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>20</y>
      <width>221</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>20</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Личный кабинет</string>
    </property>
   </widget>
   <widget class="QFrame" name="frame">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>70</y>
      <width>421</width>
      <height>71</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::WinPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Sunken</enum>
    </property>
    <widget class="QLabel" name="admin_name">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>20</y>
       <width>181</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>14</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Администратор</string>
     </property>
    </widget>
    <widget class="QTextBrowser" name="admin_name_text">
     <property name="geometry">
      <rect>
       <x>180</x>
       <y>20</y>
       <width>211</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>12</pointsize>
      </font>
     </property>
    </widget>
   </widget>
   <widget class="QFrame" name="frame_2">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>220</y>
      <width>301</width>
      <height>71</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::WinPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Sunken</enum>
    </property>
    <widget class="QPushButton" name="pushButton">
     <property name="geometry">
      <rect>
       <x>70</x>
       <y>10</y>
       <width>171</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>Начать запись пользователей</string>
     </property>
    </widget>
    <widget class="QPushButton" name="exit_button">
     <property name="geometry">
      <rect>
       <x>70</x>
       <y>40</y>
       <width>171</width>
       <height>21</height>
      </rect>
     </property>
     <property name="acceptDrops">
      <bool>false</bool>
     </property>
     <property name="text">
      <string>Выход из администратора</string>
     </property>
     <property name="checkable">
      <bool>false</bool>
     </property>
     <property name="autoRepeat">
      <bool>false</bool>
     </property>
     <property name="autoExclusive">
      <bool>false</bool>
     </property>
    </widget>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
